File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.java
import java.io.*;
import java.util.*;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int[] line1 = new int[26];
        int[] line2 = new int[26];
        String s1 = br.readLine();
        String s2 = br.readLine();
        for (int i = 0; i < s1.length(); i++) {
            line1[s1.charAt(i) - 'a']++;
        }
        for (int i = 0; i < s2.length(); i++) {
            line2[s2.charAt(i) - 'a']++;
        }
        int sum = 0;
        for (int i = 0; i < 26; i++) {
            sum += Math.abs(line1[i] - line2[i]);
        }
        System.out.println(sum);
    }
}

File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();
    let mut line1 = [0; 26];
    let mut line2 = [0; 26];
    let mut sum = 0;
    for i in input.chars() {
        if i == '\n' {
            break;
        }
        line1[i as usize - 'a' as usize] += 1;
    }
    input.clear();
    io::stdin().read_line(&mut input).unwrap();
    for i in input.chars() {
        if i == '\n' {
            break;
        }
        line2[i as usize - 'a' as usize] += 1;
    }
    for i in 0..26 {
        sum += (line1[i] - line2[i] as isize).abs() as usize;
    }
    println!("{}", sum);
}



